opentelemetry-sdk/src/metrics/data/mod.rs

```diff
@@ -9,7 +9,7 @@ use crate::Resource;
 use super::Temporality;
 
 /// A collection of [ScopeMetrics] and the associated [Resource] that created them.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
This could be refactored out! We should not have to pass Resource to exporters during each export. It can be sent just once at build time. Exporters just need scope+aggregated_metrics.
Logs already follow this model of sending Resource once. We can (and we should) follow it for Metrics and Traces too.
(a separate refactor not this PR, to keep scope contained)